### PR TITLE
Chaining proteinfold

### PR DESCRIPTION
**Pull request by luisas**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/multiplesequencealign/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/multiplesequencealign _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,9 +76,8 @@ FAMSA, -gt upgma -medoidtree, FAMSA,
 FAMSA,,REGRESSIVE,
 ```
 
-:::note
-The only required input is aligner.
-:::
+> [!NOTE]
+> The only required input is aligner.
```
Comment (JoseEspinosa) [inline README.md:80]:
```suggestion
> The only required input is `aligner`.
```

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
As we discussed I won't use the word `dependencies` but something such as `optional_data` since the work dependencies seems to suggest that this data is needed to run the pipeline. Otherwise, just small suggestions

File: docs/usage/chaining_with_proteinfold.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+nextflow run nf-core/multiplesequencealign --input ./samplesheet.csv \
+                                        --tools ./toolsheet.csv \
+                                        --dependencies_dir ./proteinfold_results/*/*/top_ranked_structures \
+                                        --outdir ./results \
+                                        -profile <docker/singularity/podman/shifter/charliecloud/conda/institute>
+
+```
+
+> [!NOTE]
+> The one imporant parameter NOT to forget in proteinfold for the chaining is `--split_fasta`. This will allow to use a multifasta file as input for monomer predictions, needed by the MSA pipeline. Also, currently the changes needed for the chaining are only present in the dev branch of proteinfold, so also do not forget `-r dev`. The rest of the proteinfold parameters can and should be tuned according to your preferences for your proteinfold run. Please refer to the proteinfold documentation for this.
```
[the remark this answers is not in the document]
Comment (luisas) [reply docs/usage/chaining_with_proteinfold.md:46]: Yess

[the remark this answers is not in the document]
Comment (luisas) [reply docs/usage/chaining_with_proteinfold.md:46]: added now

File: nextflow.config [hunk 3]
```diff
@@ -13,6 +13,7 @@ params {
     input                       = null
     tools                       = null
     templates_suffix            = ".pdb"
+    dependencies_dir            = null
```
[the remark this answers is not in the document]
Comment (luisas) [reply nextflow.config:16]: Okii changed!

**Pull request merged**